sktime/markov/transition_counting.py

```diff
@@ -560,18 +560,7 @@ def fit(self, data, *args, **kw):
         # Compute count matrix
         count_mode = self.count_mode
         lagtime = self.lagtime
-        if count_mode == 'sliding' or count_mode == 'sliding-effective':
-            count_matrix = msmest.count_matrix(dtrajs, lagtime, sliding=True, sparse_return=self.sparse)
-            if count_mode == 'sliding-effective':
-                count_matrix /= lagtime
-        elif count_mode == 'sample':
-            count_matrix = msmest.count_matrix(dtrajs, lagtime, sliding=False, sparse_return=self.sparse)
-        elif count_mode == 'effective':
-            count_matrix = msmest.effective_count_matrix(dtrajs, lagtime)
-            if not self.sparse and issparse(count_matrix):
-                count_matrix = count_matrix.toarray()
-        else:
-            raise ValueError('Count mode {} is unknown.'.format(count_mode))
+        count_matrix = TransitionCountEstimator.count(count_mode, dtrajs, lagtime, sparse=self.sparse)
```

Comment:
woot

Reply:
moved the functionality to a static method so that simple count models can be created without having to create an estimator really.

Reply:
this was like a celebratory "woot"

Reply:
oh :rocket: